core/embed/firmware/bootloader_hashes.py

```diff
@@ -19,5 +19,5 @@
     h_ff = blake2s(data=data_ff).digest()
     h_00 = "".join(["\\x%02x" % i for i in h_00])
     h_ff = "".join(["\\x%02x" % i for i in h_ff])
-    print("    // %s (padded with 0x00)\n    if (0 == memcmp(hash, \"%s\", 32)) return sectrue;" % (fn, h_00))
-    print("    // %s (padded with 0xff)\n    if (0 == memcmp(hash, \"%s\", 32)) return sectrue;" % (fn, h_ff))
+    print(f"    // {fn!s} (padded with 0x00)\n    if (0 == memcmp(hash, \"{h_00!s}\", 32)) return sectrue;")
```

Comment:
In these two lines mypy reported `On Python 3 '{}'.format(b'abc') produces "b'abc'", not 'abc'; use '{!r}'.format(b'abc') if this is desired behavior  [str-bytes-safe]` if `h_00` or `h_ff` did not have explicit `!r` or `!s`. Do I remove explicit `!s` or what do you think?

Reply:
i'm pretty sure mypy is wrong here, because it dislikes reassigning to different types: `h_00` and `h_ff` start out as `bytes` but then are turned into `str`

anyway, please revert changes in this file also, it's not worth touching for the f-string conversion so let's keep the churn to a minimum

Reply:
Reverted